package.json

```diff
@@ -20,7 +20,10 @@
     "requireindex": "~1.1.0"
   },
   "devDependencies": {
-    "eslint": ">=4.18.2",
+    "eslint": "^7.21.0",
```

Comment:
Wouldn't this version conflict with older react-scripts versions? (I have no idea what the answer really is xD)